rfcs/text/0023-match_only_text-data-type.md

```diff
@@ -28,6 +24,60 @@ The following fields are currently indexed as `text` and are candidates to migra
 * `message`
 * `error.message`
 
+ECS also has `text` type multi-fields for several fields using the convention `<field-name>.text`. This RFC also proposes to migrate all `text` multi-fields to using `match_only_text`, but the multi-field will remain using the `<field-name>.text` to avoid a breaking change.
+
+As part of this migration,
+
+* `client.as.organization.name.text`
+* `client.user.full_name.text`
+* `client.user.name.text`
+* `destination.as.organization.name.text`
+* `destination.user.full_name.text`
+* `destination.user.name.text`
+* `error.stack_trace.text`
+* `file.path.text`
+* `file.target_path.text`
+* `host.os.full.text`
+* `host.os.name.text`
+* `host.user.full_name.text`
+* `host.user.name.text`
+* `http.request.body.content.text`
+* `http.response.body.content.text`
+* `observer.os.full.text`
+* `observer.os.name.text`
+* `organization.name.text`
+* `process.command_line.text`
+* `process.executable.text`
+* `process.name.text`
+* `process.parent.command_line.text`
+* `process.parent.executable.text`
+* `process.parent.name.text`
+* `process.parent.title.text`
+* `process.parent.working_directory.text`
+* `process.title.text`
+* `process.working_directory.text`
+* `server.as.organization.name.text`
+* `server.user.full_name.text`
+* `server.user.name.text`
+* `source.as.organization.name.text`
+* `source.user.full_name.text`
+* `source.user.name.text`
+* `threat.technique.name.text`
+* `threat.technique.subtechnique.name.text`
+* `url.full.text`
+* `url.original.text`
+* `user.changes.full_name.text`
+* `user.changes.name.text`
+* `user.effective.full_name.text`
+* `user.effective.name.text`
+* `user.full_name.text`
+* `user.name.text`
+* `user.target.full_name.text`
+* `user.target.name.text`
+* `user_agent.original.text`
+* `user_agent.os.full.text`
+* `user_agent.os.name.text`
+* `vulnerability.description.text`
```

Comment:
Out of curiosity, why are we listing all multi-field usage, and no direct usage such as `message` and `error.message`?

Reply:
They are listed above the multi-fields: https://github.com/ebeahan/ecs/blob/rfc/0023/stage-1/rfcs/text/0023-match_only_text-data-type.md#fields.

I've added section headers to distinguish the two sections.